tests/data/dimension_associations/subsector__metric.csv

```diff
@@ -0,0 +1,131 @@
+subsector,metric
+FullServiceRestaurant,cooling
+FullServiceRestaurant,heating
```

Comment:
Why do you have associations in dsgrid and not dsgrid-test-data?

Reply:
dsgrid-test-data has toy versions of projects and dataset that comprise real registries. This is just a few independent files to test the DimensionAssociation class by itself.